docs/dashboards/index.md

```diff
@@ -0,0 +1,39 @@
+title: Sematext Dashboards
+description: Sematext Cloud is a modern monitoring, log management, transaction tracing, and real user monitoring system. Dashboards are a powerful feature for cohesive data analysis from multiple data sources such as logs, metrics, and events.
+
+Dashboards are your central location where you put everything together. Logs, application performance metrics, infrastructure inventory and processes, cluster data with Docker and Kubernetes metrics, and events that give you insight into what is happening with your system in real-time. Everything in one place. 
+
+Dashboards consist of multiple components that accumulate data from multiple data sources such as logs, metrics, infrastructure, and events in one central location. You can create specific reports with various metrics, correlate them with logs and event timelines to get to the bottom of issues much easier.
+
+Log data is hard to analize and can be extremely tedious to understand. Dashboards provide a means to display one or more metrics in parallel with logs, and display them in a unified report for easy correlation and investigation.
+
+## Creating Dashboards
+To create a Dashboard, start by going to the Dashboards screen and click the add new dashboard button. Give your Dashbord a name and click save.
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://sematext.com/wp-content/uploads/2019/07/create-dash.mp4" type="video/mp4" />
+</video>
+
+Follow along the [Quick Start](./quick-start) guide for more detailed instructions.
+
+## Dashboard Settings
+Dashboard settings include naming, editing, and cloning dashboards, and handling reports and components. You can choose to add any type of component to a dashboard, add, edit and clone reports and manage filters.
```

Comment:
This sentence made me think that maybe we need a Glossary.  What is a report?  What is a component?  What are filters (tags?).  Maybe we need a Glossary page, or /docs/glossary with one page for each thing we want to describe.  For those things that we cover in more detail on some page, we can link there.

Reply:
Yeah agreed. The only issue I have is when I think about it in more detail, I get the: _"if we need a glossary, our product is not simple enough, and people won't bother"_ feeling. :disappointed: 

But yes, we need it at the moment, _right now_, to describe key parts of the product. I'll add a ticket in Jira.

Reply:
https://sematext.atlassian.net/browse/IN-5264